src/index.html

```diff
@@ -1,13 +1,51 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <article class="card" data-qa="card">
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina
+        5K Monoblock (MNED2UA/A)
+      </h2>
+
+      <p class="product-code">
+        Product code: 195434
+      </p>
+
+      <div class="review">
+        <p class="review__text">Reviews: 5</p>
+        <div class="stars stars--4">
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+          <span class="stars__star"></span>
+        </div>
+      </div>
```

Comment:
also I see that you can't pass tests because of this block. so let's try to swap `stars` and `review__text` places and remove [this](https://github.com/mate-academy/layout_product-cards/pull/3425/files#diff-efc436905cb807e5305d392062b3125e8ef099b2665dfbb262301b5e1355a96eR70) property